pandas/core/arrays/interval.py

```diff
@@ -200,6 +201,9 @@ class IntervalArray(IntervalMixin, ExtensionArray):
     ndim = 1
     can_hold_na = True
     _na_value = _fill_value = np.nan
+    _left: np.ndarray
+    _right: np.ndarray
```

Comment:
I'm not sure about the type of `_left` and `_right`, see mypy error for `_from_sequence` in this file.